tests/controllers/controller_with_throttling.py

```diff
@@ -1,26 +1,6 @@
-import winter.http
 import winter
 
 
-@winter.http.throttling('6/s', scope='ControllerWithThrottlingOnController')
-@winter.no_authentication
-@winter.route_get('with-throttling-on-controller/')
-class ControllerWithThrottlingOnController:
-
-    @winter.route_get()
-    def simple_method(self):
-        return 1
-
-    @winter.route_get('same/')
-    def same_simple_method(self):
-        return 1
-
-    @winter.http.throttling(None)
-    @winter.route_post()
-    def method_without_throttling(self):
-        return None
-
-
 @winter.route_get('with-throttling-on-method/')
```

Comment:
Let's shorten the route to `with-throttling` or just `throttling` since there is no other places to put throttling on.